src/components/sections/ProductGallery/product-gallery.scss

```diff
@@ -6,8 +6,16 @@
   height: 100%;
   padding-top: 0;
 
+  [data-fs-empty-state] {
+    margin: var(--fs-spacing-3) 0;
+  }
+
   @include media(">=notebook") {
     padding-top: var(--fs-spacing-5);
+
+    [data-fs-empty-state] {
+      margin: var(--fs-spacing-5) 0;
```

Comment:
Removed the margin from the `EmptyState` component since I believe they shouldn't be allowed to set it, only their parents.

![CleanShot 2022-05-02 at 15 05 22](https://user-images.githubusercontent.com/381395/166300906-14ffacb7-2a3d-49d4-9c94-8aef43369cda.png)

This fixes the unneeded gap when the component is used inside the cart.
Before|After
-|-
![Screen Shot 2022-05-02 at 15 08 03](https://user-images.githubusercontent.com/381395/166301327-5d18aaa1-cd0c-47c8-b7b6-c94c3a0c8170.png)|![Screen Shot 2022-05-02 at 15 07 31](https://user-images.githubusercontent.com/381395/166301331-223b849d-35ac-4482-9a64-d9f74d1055e4.png)